Desktop/widgets/boundcontrolmeasurescells.cpp

```diff
@@ -77,8 +75,5 @@ void BoundControlMeasuresCells::updateOption()
 	const Terms& terms = _measuresCellsModel->terms();
 	
 	if (_boundTo)
-	{
-		_boundTo->setShouldEncode(true);
```

Comment:
Where did this go then? Superclass?

Reply:
this is set automatically because this control has a source that contains variables.